Dockerfile.microshift

```diff
@@ -0,0 +1,16 @@
+# This dockerfile is specific to building Multus for OpenShift
+FROM registry.ci.openshift.org/ocp/builder:rhel-9-golang-1.21-openshift-4.16 AS rhel9
+ADD . /usr/src/plugins
+WORKDIR /usr/src/plugins
+ENV CGO_ENABLED=1
+RUN ./build_linux.sh
+WORKDIR /
+
+FROM registry.ci.openshift.org/ocp/4.16:base-rhel9
+RUN mkdir -p /usr/src/plugins/bin
+COPY --from=rhel9 /usr/src/plugins/bin/* /usr/src/plugins/bin/
```

Comment:
Do you copy all cni plugins which is not supported in microshift? Just confirmation.

Reply:
Well, yes. Initially I thought that we can have all of them in the image and just copy onto the disk the ones we support, but now I think this wastes a bit of space - I'll change it, thx!